docs/tutorial/agent-on-docker.md

```diff
@@ -0,0 +1,157 @@
+# Deploy Vald Agent on Docker
+
+Vald is designed and implemented based on Cloud-Native architecture.
+However, there may be cases that want to use only Vald Agent without Kubernetes.
+
+This article will show you how to deploy and run the Vald Agent on Docker.
+Fashion-mnist is used as an example dataset, same as [Get Started](../tutorial/get-started.md).
+
+## Requirements
+
+- docker: v19.0 ~
+- go: v1.14 ~
+- libhdf5 (_only required for this tutorial._)
+
+
+Hdf5 is required for this tutorial. If hdf5 is not installed, please install [hdf5](https://www.hdfgroup.org/).
+<details><summary>[Optional] Install hdf5</summary><br>
+
+```bash
+# yum
+yum install -y hdf5-devel
+
+# apt
+apt-get install libhdf5-serial-dev
+
+# homebrew
+brew install hdf5
+```
+</details>
+
+## Deploy
+
+This chapter will show you how to deploy Vald Agent on docker.<br>
+This chapter will use NGT for the core engine of Vald Agent.
+
+1. Clone the vdaas/vald repository
+
+    ```bash
+    git clone https://github.com/vdaas/vald.git
+    ```
+
+1. Create `config.yaml`
+
+    The configuration of Vald agent for docker is set using `config.yaml`<br>
+    You can also check [the sample configuration](https://github.com/vdaas/vald/blob/master/cmd/agent/core/ngt/sample.yaml).
+
+    ```bash
+    cat << EOF > config.yaml
+    ---
+    version: v0.0.0
+    time_zone: JST
+    logging:
+      logger: glg
+      level: debug
+      format: json
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
How about to use 'raw' format for logging?